Makefile

```diff
@@ -63,6 +64,17 @@ start-FULL-REBUILD: echo_vars stop rm-ALL ## Remove and restart all Docker conta
 	docker-compose ${COMPOSE_FILE_ARGS} --env-file ${ENV_FILE} down
 	docker-compose ${COMPOSE_FILE_ARGS} --env-file ${ENV_FILE} up --build
 
+extract-bundles: ## Extract bundles from file-server for cloud deployment
```

Comment:
suggest either `static-assets`, `client-assets` or some such vs `bundles` for clarity, since the meaning of "bundles" is less universal.